app/js/app-menu.js

```diff
@@ -32,6 +32,32 @@ module.exports = function() {
       }
     });
   }
+
+  function cd2sbgnml(xml) {
+
+    $.ajax({
+        type: 'post',
+        url: "http://localhost:4567/cd2sbgnml",
+        data: {xml: xml},
+        success: function (data) {
+            var chiseInstance = appUtilities.getActiveChiseInstance();
+            validateSBGNML(xml);
+            chiseInstance.loadSBGNMLText(data);
```

Comment:
@gokberkkaraca 
- spinner should be started before loading and stopped after loading
-  `sbgnvizLoadFileStart` and `sbgnvizLoadFileEnd` events should be triggered when loading starts and finishes, [see this](https://github.com/iVis-at-Bilkent/sbgnviz.js/blob/205fad5a9853d31d2372b9cb2525662b8251bcf0/src/utilities/file-utilities-factory.js#L182)